packages/wrangler/src/__tests__/helpers/msw/handlers/deployments.ts

```diff
@@ -1,7 +1,7 @@
 import { rest } from "msw";
 
 export type DeploymentListRes = {
-	versions: {
+	deployments: {
```

Comment:
This will be `items` instead, context is inferred from URL.

Reply:
Additionally, each item does not have `preview` or `resources` anymore, they will be shortened.